js/framework/AnalyticsConstants.js

```diff
@@ -164,6 +164,15 @@ if (!OO.Analytics.EVENTS)
 
     /**
      * @public
+     * @event OO.Analytics.EVENTS#VIDEO_STREAM_POSITION_CHANGED
+     * @description This message is sent when the video stream position changes.
```

Comment:
can you add that it's sent 'periodically'? So it's not expected to update every 16ms or 32ms (thats 60 or 30 fps)